api_v2/src/leaves/leaves.service.ts

```diff
@@ -230,4 +239,38 @@ export class LeavesService {
       leave.leaveProjects,
     );
   }
+
+  public leaveDaysToHours(days: number): number {
```

Comment:
Just a suggestion maybe we can do it like this for better readability
```suggestion
  private readonly leaveConversionMap = new Map<number, number>([
  [0.125, 4],
  [0.19, 1.5],
  [0.25, 2],
  [0.31, 2.5],
  [0.38, 3],
  [0.5, 4],
  [0.625, 5],
  [0.69, 5.5],
  [1, 8],
]);

public leaveDaysToHours(days: number): number {
  const rounded = parseFloat(days.toFixed(3)); // To avoid floating-point quirks
  return this.leaveConversionMap.get(rounded) ?? 0;
}
```